pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Run command:
```

Comment:
This example is placed at the top, with the expectation that most of the time the user will be running commands